contribute/editorial_board_guide.md

```diff
@@ -0,0 +1,222 @@
+
+  * Check open issues regularly or enable notifications by clicking the "WATCH" icon in the top-right side of the [GitHub repository](https://github.com/elixir-europe/infectious-diseases-toolkit/issues).
+  * Assign labels to issues.
+  * Discuss who is going to be responsible for each issue with other editors and reviewers (via issue comments or other communication channels).
+  * Assign at least one editor/reviewer to the issue, who will discuss the possible content with the contributor.
+  * When a Pull Request (PR) or a draft PR related to an issue is created, link the PR to the issue.
+  
+More information about these topics can be found in the GitHub documentation:
+- [commenting on PRs](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)
+- [start a review](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request)
+
+## Review pull requests
+
+If contributors make a pull request to make changes, by default the editors that are responsible for files that will changed by the PR will be assigned and notified. All PR should be assigned to one of the editors. Before merging a PR, pages' tags and keywords, and tools and resources' tags should be checked and assigned according to the established tagging system.
+  
+## Link a pull request to an issue
+
+When you make a pull request resolving an issue, it is possible to link this pull request to that specific issue. This can be easily done by writing in the conversation of the PR: `closes #number_of_issue`, or `fixes #number_of_issue` or even `resolves #number_of_issue`. This is definitely applicable when authors first open an issue announcing a change or requesting a new page, followed up by the pull request. 
+For more information about this topic please visit the [GitHub documentation page](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
+
+## Adding a new event
+
+Add an event to the landing page by editing the `events.yml` in the `_data` directory in this repository. Use following attributes to define an event:
+
+
+```yml
+- name: Example event
+  startDate: 2021-06-23
+  startTime: '9:00'
+  endDate: 2021-06-24
+  endTime: 13:30 CET
+  description: Example description
+  location: Online
+
+```
+
+Only name and startDate are mandatory attributes.
+
+## Adding a news item
+
+Add a news item to the landing page by editing the `news.yml` in the `_data` directory in this repository. Use following attributes to define a news item:
+
+
+```yml
+- name: News title
+  date: 2021-06-23
+  linked_pr: 767
+  description: A short description
+```
+
+All attributes are mandatory.
+
+## Create a new page
+
+### Simple way: using the GitHub interface
+To generate a new page it is sufficient to simply copy the TEMPLATE file in the subdirectory and rename it. To copy a template you have to:
```

Comment:
```suggestion
To generate a new page it is sufficient to simply copy the TEMPLATE file to the desired subdirectory and rename it. To copy a template you have to:
```
Tried to clarify that the template file needs to be copied to the correct subdirectory, and that the original template file to be copied is not found at the subdirectory